ssc/cmod_battery_eqns.cpp

```diff
@@ -108,3 +109,157 @@ bool Calculate_thermal_params(ssc_data_t data) {
     return true;
 }
 
+bool Reopt_size_standalone_battery_params(ssc_data_t data) {
+    auto vt = static_cast<var_table*>(data);
+    if (!vt) {
+        return false;
+    }
+    std::string log;
+    auto reopt_params = var_data();
+    reopt_params.type = SSC_TABLE;
+    var_table* reopt_table = &reopt_params.table;
+    var_table reopt_scenario, reopt_site, reopt_electric, reopt_utility, reopt_load, reopt_fin, reopt_batt,
+        reopt_wind;
+    reopt_wind.assign("max_kw", 0);
+
+    var_data* vd, * vd2;
+
+    double val1, val2, system_cap;
+    vt_get_number(vt, "system_capacity", &system_cap);
+
+    // financial inputs
+    map_optional_input(vt, "itc_fed_percent", &reopt_batt, "total_itc_pct", 0., true);
+    // TODO: what about reopt vars total_rebate_us_dollars_per_kw?
```

Comment:
What about this TODO? Are there any incentives?

Reply:
That TODO has been in there for a while and got copy-pasted over. I'm thinking this TODO might be a better fit for a larger REopt API version upgrade closer to release time, the latest version doesn't include that variable: https://nrel.github.io/REopt.jl/dev/reopt/inputs/#Financial